app/routines/openstax/accounts/find_or_create_account.rb

```diff
@@ -7,7 +7,8 @@ class FindOrCreateAccount
       protected
 
       def exec(email: nil, username: nil, password: nil,
-               first_name: nil, last_name: nil, full_name: nil, title: nil)
+               first_name: nil, last_name: nil, full_name: nil, title: nil,
+               salesforce_contact_id: nil, faculty_status: nil)
```

Comment:
I don't think we want SF contact ID and faculty status here, because we don't want to be able to create an account where we set those fields -- those fields are only ever set via a pull from Salesforce, inside Accounts.  Is there another use case I'm missing?